.travis.yml

```diff
@@ -1,6 +1,5 @@
+sudo: false
 language: node_js
 node_js:
   - "4.1"
-before_script:
-  - npm install -g pawelgalazka/runjs
-script: run test
\ No newline at end of file
+script: "npm run build && npm test"
```

Comment:
We could just leave `before_script: npm run build` and `npm test` would be picked up by default.